src/scenes/Event/scenes/oracle.js

```diff
@@ -690,6 +689,7 @@ export default class OraclePage extends React.Component {
     const { oracle } = this.state;
     const totalAmount = _.sum(oracle.amounts);
     const { locale, messages: localeMessages } = this.props.intl;
+    window.oracle = oracle;
```

Comment:
is this normal to have the window hold the var?

Reply:
ahhh! Sorry. Totally debugging and forgot to remove. Good catch